styles.css

```diff
@@ -96,10 +102,28 @@ section {
         list-style: none;
         margin-right: 10px;
     }
-    .nav-buttons {
-        width: 10%;
+
+    .footer-links {
+        overflow: hidden;
+        position: relative;
+        display: inline-block;
+        width: 100%;
+        height: 30vh;
     }
-}
+    .footer-nav-bar {
+        float: left;
+        margin: 0 auto;
+        width: 100%;
+        /*margin-top: 50px;*/
```

Comment:
remove unused code before sending to Charlie?

Reply:
removed